cc/SLIIT/2022-current.md

```diff
@@ -0,0 +1,36 @@
+<img src="https://github.com/sliit-foss/brand-assets/raw/master/Logo/Logo.jpg" width="20%" />
+
+# SLIIT - Sri Lanka Institute of Information Technology
+
+A group of volunteers who believe in the usage of Free/Open Source Software (FOSS). The primary objective of the community is to promote, develop and diversify the usage of Free and Open Source Software at SLIIT.
+
+## Members
+
+- **Nisal Renuja Palliyaguru (Club Lead)** [@Linkedin](https://www.linkedin.com/in/nisalrenuja/)
+- Madhusha Prasad [@Linkedin](https://www.linkedin.com/in/madhusha-prasad-045a82187/)
+- Sahas Gunasekara [@Linkedin](https://www.linkedin.com/in/5ahas/)
+- Methmi Nugawela [@Linkedin](https://www.linkedin.com/in/methmi-nugawela/?originalSubdomain=lk)
+- Lakisuru Semasinghe [@Linkedin](https://www.linkedin.com/in/lakizuru/)
+- Bhagya Indimagedara [@Linkedin](https://www.linkedin.com/in/bhagya-indimagedara/)
+- Miyuru Gunarathna [@Linkedin](https://www.linkedin.com/in/miyurugunarathna/)
+- Akalanka Perera [@Linkedin](https://www.linkedin.com/in/akalanka-perera-50899a1b3/)
+- Thushan Jaliya [@Linkedin](https://www.linkedin.com/in/thushan-jaliya-3b1507202/)
+- Danuja Hemal [@Linkedin](https://www.linkedin.com/in/danuja-jayasuriya-11483b211/)
```

Comment:
No need to include LinkedIn here. Please proceed according to the template given.